### DISCUSSION TITLE
[blocks]: Simple DND form builder

### DISCUSSION THREAD

**Discussion by bharath917** [shadcn-ui/ui#3186] [Blocks Request]
### Description

A simple DND form builder, there are currently no good open-source versions.

### Example

https://builder.vueform.com/demo

**Comment by iduspara** [comment]
I'm currently building a modern form builder application that allows you to create complex forms with a drag-and-drop interface. This is a fun side project I work on in my free time, and while I'm not full-time on it, I'm actively developing new features and improvements when I can.

I would like to get a feedback from the community so we can improve it together...

You can check it out here: [https://www.shadcn-builder.com/](https://www.shadcn-builder.com/?utm_source=github&utm_content=shadcn-u-discussion-3186&ref=shadcn-u-discussion-3186)
